docs/docs/classic-ml/search/search-models/index.mdx

```diff
@@ -0,0 +1,525 @@
+import { APILink } from "@site/src/components/APILink";
```

Comment:
Similar comment in the `search` folder, could we directly use `search-experiments.mdx` etc instead of creating a folder for each part?